docs/_includes/partials/component/back-to-top.njk

```diff
@@ -1,45 +1,2 @@
-<style>
-  /* TODO: Replace back-to-top with RHDS component */
-  /* unique id placed due to javascript converting headers to ids */
-  #back-to-top-q3x4zkd {
-    display: none;
-    pointer-events: none;
-    position: sticky;
-    justify-content: flex-end;
-    width: 100%;
-    bottom: 32px;
-    margin-block-start: -64px;
-    padding-inline-end: 32px;
-  }
```

Comment:
we should reapply these custom styles so the back to top displays as you scroll rather then just when you reach the bottom of the page.  

```
rh-back-to-top {
  display: flex;
  pointer-events: all;
  position: sticky;
  justify-content: flex-end;
  width: 100%;
  bottom: 32px;
  margin-block-start: -64px;
  padding-inline-end: 32px;
 }
```

The above snippet seems to do it.

Reply:
the element doesn't do that on it's own?

Reply:
I would call the above specific to the website.  Depends on how you want to position it on the page.  The component will hide and show based on scroll but doesn't apply custom positioning.  `rh-back-to-top` is pretty much a copy / paste duplicate from `pf-back-to-top`.  I think we have room for improvement there.